src/AutoRest.CSharp/Common/AutoRest/Plugins/NewProjectScaffolding.cs

```diff
@@ -333,7 +188,6 @@ private string GetBrandedSrcCSProj()
 
             return builder.Write();
         }
-
```

Comment:
it would be better to keep new line between methods

Reply:
Fixed